include/client.h

```diff
@@ -31,22 +31,58 @@ namespace mixer_client {
 // is completed.
 using DoneFunc = std::function<void(const ::google::protobuf::util::Status&)>;
 
+// Define transport context status
+typedef enum {
+  CTX_OK = 0,
+  CTX_NOT_EXIST,
+} CtxStatus;
+
+// Transport Context ID type.
+typedef int CtxID;
```

Comment:
@geeknoid GRPC runtimes and H2 proxies can and do limit the number of streams so we need to be cognizant of that. GRPC will buffer streams it cannot yet create but it can do this silently which may not be what we desire.

This can be compensated for by having more connections but it will need some thought if we expect to go over the default stream count limit of 100.